### ISSUE TITLE
Feature request: Add a layer of air to avoid some failures

### ISSUE THREAD

**Issue by Yxmna** [rebane2001/mapartcraft#207]
Hi, 
I would like you to add, in the schematica, an extra layer of air block above the map-art
Why ?
Sometimes I unintentionally put carpets on top of a block:
![](https://cdn.discordapp.com/attachments/822787638615474176/948234136500457532/2022-03-01_15.26.48.png)
Like this light gray carpet above a stone that makes it almost invisible

The extra layer of air would simply allow litematica to warn us of this type of error ^^

**Comment by nyncral** [comment]
yes, and it's better to have a layer of air above (that would be useful) than 2 layers of air under it that just annoy some people
